src/components/DataTable.vue

```diff
@@ -33,16 +33,25 @@
 </template>
 
 <script>
+import MoneyComponent from '@/components/MoneyComponent'
+
 export default {
     name: "DataTable",
 
+    components: {
+      MoneyComponent
+    },
+
     props: {
       bikes: Array
     },
 
     data() {
       return{
         search: '',
+
+        styling: 'text-xs-right',
+
         header:[ 
           
           { text: 'Navn', align: 'left', value: 'title' },
```

Comment:
```suggestion
          { text: 'Name', align: 'left', value: 'title' },
```

Reply:
I am not sure, but didn't we agree to have it all in danish? or are we going for a english version?